index.bs

```diff
@@ -187,20 +187,62 @@ partial interface Navigator {
 
 ## ML ## {#api-ml}
 <script type=idl>
+// WebGL specification:
+// (https://www.khronos.org/registry/webgl/specs/latest/2.0/#2.1)
+[SecureContext, Exposed=Window]
+interface WebGLRenderingContext {
+  // ...
+};
+
+// WebGPU specification:
+// (https://gpuweb.github.io/gpuweb/#gpu-device)
+[SecureContext, Exposed=Window]
+interface GPUDevice {
+  // ...
+};
```

Comment:
We should use Bikeshed's [custom definitions](https://tabatkins.github.io/bikeshed/#custom-dfns) for adding anchors to other IDL that is not yet part of the autolinking database.

To do that, you can add the  following additional `<pre class="anchors">` block e.g. below the `<pre class='metadata'>` block and remove the `interface WebGLRenderingContext` and  `interface GPUDevice` declarations. Bikeshed is picky about the whitespace here, so indentation must be exactly a multiple of 4 spaces (or tabs):
```
</pre>
<pre class="anchors">
urlPrefix: https://www.khronos.org/registry/webgl/specs/latest/2.0/; spec: WEBGL-2
    type: interface
        text: WebGLRenderingContext; url: 2.1
        text: WebGLBuffer; url: 5.4
        text: WebGLTexture; url: 5.9
urlPrefix: https://gpuweb.github.io/gpuweb/; spec: WEBGPU
    type: interface
        text: GPUDevice; url: gpu-device
        text: GPUBuffer; url: buffer-interface
        text: GPUTexture; url: texture-interface
</pre>
```

Reply:
Thanks a lot @anssiko. I'm expecting exactly this type of feedback from others who are more familiar with Bikeshed than I am. Will look into this right away.

Reply:
Happy to help. For future reference, [definitions data model](https://tabatkins.github.io/bikeshed/#dfn-contract) explains what metadata is needed for cross-spec autolinking to work. If we wish to export definitions to be used by other specs, we should comply with this data model. I believe IDL definitions are automatically exported.

Reply:
Fixed. Please have a look.

Reply:
LGTM.